src/Service/Cases.php

```diff
@@ -96,4 +96,52 @@ public function getByCaseId(string $caseId, string $caseTypePrefix = null): ?arr
 
         return $cases['CasesInfo'][0] ?? null;
     }
+
+    /**
+     * Example query.
+     *
+     * $query = [
+     *   'CaseTypePrefix' => 'BOR',
+     *   'MetadataXml' =>
+     *     "<z:row xmlns:z=\"#RowsetSchema\"
+     *      ows_Title=\"0123456789 - Test borger\"
+     *      ows_CCMContactData=\"Test borger;#;#0123456789;#;#\"
+     *      ows_CCMContactData_CPR=\"0123456789\"
+     *      ows_CaseStatus=\"Åben\"
+     *      />",
+     *  'ReturnWhenCaseFullyCreated' => true,
+     * ];
+     */
+    public function CreateCase(array $query): ?array
+    {
+        return $this->getData(
+            'POST',
+            $this->getApiBasePath(),
+            ['json' => $query],
+        );
+    }
+
+    /**
+     * Example query.
+     *
+     * $query = [
+     *   'CaseTypePrefix' => 'BOR',
+     *   'MetadataXml' =>
+     *     "<z:row xmlns:z=\"#RowsetSchema\"
+     *     ows_Title=\"Undersag - test\"
+     *     ows_CCMParentCase=\"BOR-2022-000038\"
+     *     ows_ContentTypeId=\"0x0100512AABDB08FA4fadB4A10948B5A56C7C01\"
+     *     ows_CCMContactData_CPR=\"0123456789\"
+     *     ows_CaseStatus=\"Åben\"/>",
+     *  'ReturnWhenCaseFullyCreated' => true,
+     *  ];
+     */
+    public function CreateSubCase(array $query): ?array
```

Comment:
The parameter should not be called `$query`, but rather `$data` or something similar. “Query” is used for querying and not for creating stuff.

Reply:
Updated variable name.